mixer/adapter/stackdriver/contextgraph/send.go

```diff
@@ -119,9 +122,60 @@ func (h *handler) send(ctx context.Context, t time.Time, entitiesToSend []entity
 		req.RelationshipPresentAssertions = append(req.RelationshipPresentAssertions, relAsst)
 	}
 
-	// TODO: Batch requests if there are too many entities and edges in one request.
+	h.env.Logger().Debugf("Context api request size: %v", proto.Size(req))
+	h.env.Logger().Debugf("Sending %v entities and %v relationships",
+		len(req.EntityPresentAssertions), len(req.RelationshipPresentAssertions))
+
+	if proto.Size(req) < maxReq {
+		if err := h.call(ctx, req); err != nil {
+			return err
+		}
+	} else {
```

Comment:
Can we add a test to validate the batching logic?

Reply:
Done.